packages/odyssey-react/src/components/Infobox/index.tsx

```diff
@@ -72,10 +73,17 @@ const Infobox: FunctionComponent<Props> & StaticComponents = (props) => {
 
   const omitProps = useOmit(rest);
 
+  const icon = {
+    caution: <Caution />,
+    danger: <Error />,
+    info: <GetInfo />,
+    success: <Complete />
+  }
```

Comment:
ditto to my above comment. This approach also reminds me that while we have nice type enforced props we don't have any guarantees that folks *not* using typescript will know how to interact with our API so I believe we have additional work to do around adding `propType` definitions for those use cases sometime in the future